.github/workflows/workflow.yaml

```diff
@@ -50,13 +50,18 @@ jobs:
       with:
         go-version-file: go.mod
     - name: Build
-      run: make debug-workflow all
+      run: make all manifest
     - uses: google-github-actions/auth@v2 # This BREAKS the git checkout/tags/history, so must run after builds!
       with:
         credentials_json: ${{ secrets.GCP_CREDENTIALS }}
     - uses: google-github-actions/setup-gcloud@v2
-    - name: Upload binaries to GCS
-      run: gsutil -h "Cache-Control:no-cache" cp bin/viam-agent-* gs://packages.viam.com/temp/prerelease/ #TODO use real target after live testing
-    - name: "Upload install scripts to GCS"
+    - name: Upload binaries to GCS (Release)
       if: github.event_name == 'release'
-      run: gsutil -h "Cache-Control:no-cache" cp preinstall.sh install.sh uninstall.sh gs://packages.viam.com/temp/prerelease/ #TODO use real target after live testing
+      run: |
+        gsutil -h "Cache-Control:no-cache" cp bin/viam-agent-* install.sh uninstall.sh preinstall.sh gs://packages.viam.com/apps/viam-agent/
+        gsutil cp etc/viam-agent-*.json gs://packages.viam.com/apps/viam-subsystems/
```

Comment:
the second `gsutil cp` line is duplicated, could be moved to its own step so it can be shared?

only do this if it makes this clearer from your perspective though -- it's a style nit